TADashboard/src/test/java/tests/data_profiles/PositiveDPTests.java

```diff
@@ -0,0 +1,56 @@
+package tests.data_profiles;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.DashboardPage;
+import page_objects.DataProfilesPage;
+import page_objects.LoginPage;
+import tests.BaseTest;
+import utils.common.Constants;
+import utils.helper.Logger;
+
+public class PositiveDPTests extends BaseTest {
+    private String[] preSetDataProfiles = {
```

Comment:
use List instead

Reply:
Do You mean I should use List instead String array?

Reply:
I have updated for my code. Please check again.